docs/conf.py

```diff
@@ -0,0 +1,383 @@
+# -*- coding: utf-8 -*-
+#
+# Auto-PyTorch documentation build configuration file, created by
+# sphinx-quickstart on Thu May 21 13:40:42 2015.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+
+import os
+import sys
+import sphinx_bootstrap_theme
+import autoPyTorch
+# Add the parent directory of this file to the PYTHONPATH
+import os
+
+current_directory = os.path.dirname(__file__)
+parent_directory = os.path.join(current_directory, '..')
+parent_directory = os.path.abspath(parent_directory)
+sys.path.append(parent_directory)
+
+import autoPyTorch
+
+# -- General configuration ------------------------------------------------
+
+# If your documentation needs a minimal Sphinx version, state it here.
+# needs_sphinx = '1.0'
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.autosummary',
+              'sphinx.ext.doctest', 'sphinx.ext.coverage',
+              'sphinx.ext.mathjax', 'sphinx.ext.viewcode',
+              'sphinx_gallery.gen_gallery', 'sphinx.ext.autosectionlabel',
+              # sphinx.ext.autosexctionlabel raises duplicate label warnings
+              # because same section headers are used multiple times throughout
+              # the documentation.
+              'numpydoc']
+
+
+from sphinx_gallery.sorting import ExplicitOrder, FileNameSortKey
+
+# Configure the extensions
+numpydoc_show_class_members = False
+autosummary_generate = True
+
+# prefix each section label with the name of the document it is in, in order to avoid
+# ambiguity when there are multiple same section labels in different documents.
+autosectionlabel_prefix_document = True
+
+# Sphinx-gallery configuration.
+
+# get current branch
+binder_branch = 'refactor_development'
+import autoPyTorch
```

Comment:
also, here?

Reply:
oh, because the python path is modified for binder. We import it after messing with the original system path. This is to enable binder, and sadly I don't know a better way to do this.

Reply:
okay, seems like this is the only choice